fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java

```diff
@@ -0,0 +1,186 @@
+package com.github.dedis.popstellar.repository.remote;
+
+import android.util.Log;
+
+import com.github.dedis.popstellar.model.network.GenericMessage;
+import com.github.dedis.popstellar.model.network.answer.Answer;
+import com.github.dedis.popstellar.model.network.answer.Error;
+import com.github.dedis.popstellar.model.network.answer.ResultMessages;
+import com.github.dedis.popstellar.model.network.method.Broadcast;
```

Comment:
how about the following, given the massive amount of imports ? 
```suggestion
import com.github.dedis.popstellar.model.network.answer.*;
import com.github.dedis.popstellar.model.network.method.*;
```

Reply:
I would need to change the import auto-formatting